ideascube/static/ideascube/vendor/purecss/README.md

```diff
@@ -0,0 +1,177 @@
+Pure
```

Comment:
Do we really need to bundle their big `README.md` file, full of info we don't really care about?

I would prefer a simpler file we'd write, containing just a few info about what we've bundled:

* upstream URL
* version bundled
* download URL